arbos/tx_processor.go

```diff
@@ -335,13 +339,13 @@ func (p *TxProcessor) EndTxHook(gasLeft uint64, success bool) {
 		refund := arbmath.BigMulByUint(gasPrice, gasLeft)
 
 		// undo Geth's refund to the From address
-		err := util.TransferBalance(&inner.From, nil, refund, p.evm, util.TracingAfterEVM)
+		err := util.TransferBalance(&inner.From, nil, refund, p.evm, scenario, "ignore")
```

Comment:
Maybe call this `undoRefund`? It can't be ignored as it is a transfer.

Reply:
Oops, I thought this was during evm execution and that it didn't matter. It's now `underRefund`